CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Changelog
 
 ## 2022-05-04
-* Updated stackstorm-ssh volume mount path in docker-compose.yml st2actionrunner service
+* Upgrade MongoDB `v4.0` -> `v4.4` as 4.0 has reached its EOL. (#243)
+* Fix stackstorm-ssh volume mount path in docker-compose.yml st2actionrunner service (#244)
```

Comment:
Is this a part of this PR? Okay to leave this here I guess if it was missed in previously merged PRs though.

Reply:
Yeah, it just fixes the changelog for the previous record.